src/fundamental_interface.jl

```diff
@@ -412,6 +412,13 @@ The two argument version is a shorthand for `div(a, a, b)`.
 div!(z, a, b) = AbstractAlgebra.div(a, b)
 div!(a, b) = div!(a, a, b)
 
+@doc """
+    divides!(z, a, b)
+
+Return `divides(a, b)`, possibly modifying the object `z` in the process.
+"""
+divides!(z, a, b) = divides(a, b)
```

Comment:
While I am happy to accept the PR as-is, I just wanted to point out that in other `!` functions, the first argument is usually returned... For `addmul` we have `addmul(z, a, b, t)` with `t` used as temporary storage. So perhaps this should be
```suggestion
divides!(a, b, t) = divides(a, b)
```

But again, I am absolutely fine with either, just wanted to point this out.

Oh,

Reply:
I disagree. divides returns two things: a Bool if the quotient exists, and the quotient of a and b. And I think the Nemo PR wants to put the quotient into z, which would make it the first arg by our convention